cmake/modules/CPackREADME.cmake

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+#---------------------------------------------------------------------------------------------------
+#  CPackREADME.cmake
+#   - Pre-packaging script to convert README.md to html
+#---------------------------------------------------------------------------------------------------
+
+if (APPLE)
+  find_program(CONVERTER textutil)
```

Comment:
Please add a comment along the lines "Apple productbuild cannot handle .md files as `CPACK_PACKAGE_DESCRIPTION_FILE`s; convert to HTML instead."